alg/contour.cpp

```diff
@@ -607,7 +610,19 @@ CPLErr GDALContourGenerateEx(GDALRasterBandH hBand, void *hLayer,
         fixedLevels.resize(aosLevels.size());
         for (size_t i = 0; i < fixedLevels.size(); i++)
         {
-            fixedLevels[i] = CPLAtof(aosLevels[i]);
+            // Handle min/max values
+            if (EQUAL(aosLevels[i], "MIN"))
```

Comment:
The Doxygen for that option (lines 517-519) should also mention "MIN" and "MAX" as valid values.

Reply:
done